bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/Edge.java

```diff
@@ -282,6 +279,95 @@ static int callAndWait(String[] pstr, ToIntFunction<IUnknown> callable) {
 	return phr[0];
 }
 
+class WebViewProvider {
```

Comment:
It would be good to use visibilities of methods in this class to indicate which methods are supposed to be executed from outside and which are not (i.e., make the former methods public/default visibility). That does not introduce encapsulation that is compiler-enforced, but at least it gives a hint for developers which methods are for internal structuring of the class and which are part of its interface to outside.

Reply:
alright